torch_np/_funcs_impl.py

```diff
@@ -1750,8 +1666,7 @@ def round_(a: ArrayLike, decimals=0, out: Optional[OutArray] = None):
     return result
 
 
-around = round_
-round = round_
+around = round
```

Comment:
so we're OK with not having a `round_` alias? IUIC it might get deprecated in NumPy 2.0, so probably OK.

Reply:
Oh, `round_` is also a thing?! Let me add that. Leaving `round` as the main one tho.